src/tennis/core.clj

```diff
@@ -0,0 +1,131 @@
+(ns tennis.core)
+
+(def config {:max-matches 3
+             :labels [0 15 30 40 "A"]})
+
+
+(def default-scores {:player1 [0 [0] 0]
```

Comment:
It would be good to add here a doc string describing what is the structure of score. To get it I had to read line 47:
```
(let [[game sets match] (player score)
```

Reply:
Made the structure of score to be a map like {:player {:point 0 :sets [0] :match 0}}. Thank you!